uptane-standard.md

```diff
@@ -354,7 +357,9 @@ Uptane does not specify implementation details. Instead, this Standard describes
 At a high level, Uptane requires:
 
 * Two software repositories:
-    * An Image repository containing binary images to install and signed metadata about those images.
+    * An Image repository containing:
+	    * binary images
+	    * signed metadata about the binary images
```

Comment:
I don't really see a strong reason to make this change.  Was this unclear as written?

Or at least if we do make this change, the director, etc. should split this all out too.

Reply:
In my first read through it was confusing, but after reading back it is not.

It has to do with thinking "install and signed" where two verbs are together sounds like it should be performed by the "binary images"

"Binary images to install and signed" is how immediately saw it.